app/src/main/java/com/pixlee/pixleeandroidsdk/ui/uploader/ImageUploaderFragment.java

```diff
@@ -119,15 +126,34 @@ public void onError(String error) {
     }
 
     // Sample 2: Upload an image using a file
-    private void uploadFile(String filePath) {
+    @Override
+    public void uploadFile(String filePath) {
         showMessage("Uploading  " + filePath);
         setUploadingUI(true);
+
+        JSONObject json = new JSONObject();
+        try {
```

Comment:
hardcodes?

Reply:
This is the demo app area and is to demonstrate how the SDK can be used. I'll replace this with a sample data.